doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -1,18 +1,605 @@
+    arr.to_excel('arrays.xlsx', 'data_only', header=False)
+
+  id: 5
+
+- markdown: |
+    #### Dumping Arrays to Excel Sheets with open_excel 
+
+
+- markdown: |
+    ##### Basic Usage
+    
+    ``open_excel`` must be used with the Python keyword ``with``. If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file. 
+    For example: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    # to create a new Excel file, argument overwrite_file must be set to True
+    with open_excel('new_excel_file.xlsx', overwrite_file=True) as wb:
+        # add a new sheet 'arr' and dump the array 'arr' in it 
+        wb['arr'] = arr.dump()
+        # add a new sheet 'arr2' and dump the array 'arr2' in it 
+        wb['arr2'] = arr2.dump()
+        # add a new sheet 'arr3' and dump the array 'arr2' in it 
+        wb['arr3'] = arr3.dump()
+        # actually write data in the Workbook
+        wb.save()
+        
+    # the Workbook is automatically closed when getting out the block 
+    # defined by the with statement
+    print('dumping arrays arr, arr2 and arr3 is done')
+
+  id: 6
+
+- markdown: |
+    ##### Specifying Position
+    
+    It is possible to choose the position of the first cell containing the data of the dumped array. This allows to export several arrays in the same sheet: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # add a new sheet 'arrays' and dump the array 'arr' starting at cell 'A1' 
+        wb['arrays'] = arr.dump()
+        # dump the array 'arr2' in sheet 'arrays' starting at cell 'A5' 
+        wb['arrays']['A5'] = arr2.dump()
+        # dump the array 'arr3' in sheet 'arrays' starting at cell 'A9' 
+        wb['arrays']['A9'] = arr3.dump()
+        wb.save()
+
+  id: 7
+
+- markdown: |
+    ##### Specifying Wide VS Narrow format
+    
+    Like with the ``to_excel`` method, it possible to export the data in a ``narrow`` format by setting the argument ``wide`` to False:
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # dump the array 'arr' in a narrow format: 
+        # one column per axis plus one value column.
+        # Argument value_name can be used to change the name of the 
+        # column containing the data (default name is 'value')
+        wb['narrow'] = arr.dump(wide=False, value_name='data')
+        wb.save()
+        
+    # in the sheet 'narrow', data are written as:
+    # a  | b  | data  |
+    # a0 | b0 | 0     |
+    # a0 | b1 | 1     |
+    # a1 | b0 | 2     |
+    # a1 | b1 | 3     |
+
+  id: 8
+
+- markdown: |
+    ##### Exporting only data
+    
+    To export only data in Excel sheets, you can set argument ``header`` of the method ``dump`` to false or not calling ``dump``:
```

Comment:
s/argument header/header argument
s/method dump/dump method/
s/false/False
s/not calling/avoid calling/